7_react/react1/week2/todo-app/src/todoList/TodoList.jsx

```diff
@@ -0,0 +1,53 @@
+import TimeCount from "../timer/Timer";
+import TodoItem from "../todoItem/TodoItem";
+import todos from "../todoItem/todos";
+import { useState } from "react";
+
+function AppTodo() {
+  // const data = [...todos];
+  const [todolist, setTodoList] = useState([...todos]);
+
+  const addTodo = () => {
+    const newTodo = {id:new Date(), name: "new todo", date: new Date(), status: false };
+    setTodoList((todolist) => { return [...todolist, newTodo]; });
```

Comment:
State setter function updates the state with the value you pass to it, so you can simplify this snippet like
`setTodoList([...todolist, newTodo])`